res/controllers/midi-components-0.0.js

```diff
@@ -818,9 +818,9 @@
             this.setCurrentUnit(this.unitNumbers[index]);
         };
 
-        if (Array.isArray(unitNumbers)) {
+        if (unitNumbers !== undefined && Array.isArray(unitNumbers)) {
```

Comment:
`this.unitNumbers` is still accessed below. How about checking `unitNumbers === undefined` once before this if/else block?

Reply:
... But it returns I line 826?

Reply:
@Be-ing ping